ConcurrentSeqNoVersioningIT#testSeqNoCASLinearizability fails with seq_no_stats mismatch

The following failed during the master intake build:

https://scans.gradle.com/s/6rusoszfwcte2/console-log?task=:server:integTest

```
org.elasticsearch.versioning.ConcurrentSeqNoVersioningIT > testSeqNoCASLinearizability FAILED
java.lang.AssertionError: [test][2], node[Kg-y0wdEQFClfDOpc0SNKQ], [R], s[STARTED], a[id=lRSFjtiaQS6zIXN4Y5NSMw] seq_no_stats mismatch
Expected: <SeqNoStats{maxSeqNo=74, localCheckpoint=74, globalCheckpoint=74}>
but: was <SeqNoStats{maxSeqNo=74, localCheckpoint=74, globalCheckpoint=73}>
REPRODUCE WITH: ./gradlew :server:integTest --tests "org.elasticsearch.versioning.ConcurrentSeqNoVersioningIT.testSeqNoCASLinearizability" -Dtests.seed=F311DE91A864DA2D -Dtests.security.manager=true -Dtests.locale=nl -Dtests.timezone=NZ-CHAT -Dcompiler.java=12 -Druntime.java=11
```

Possibly this is a relative of #42244 although the failure looks different so I'm opening a separate issue.

--- a/server/src/test/java/org/elasticsearch/versioning/ConcurrentSeqNoVersioningIT.java
+++ b/server/src/test/java/org/elasticsearch/versioning/ConcurrentSeqNoVersioningIT.java
@@ -136,6 +136,7 @@ public class ConcurrentSeqNoVersioningIT extends AbstractDisruptionTestCase {
 
         assertAcked(prepareCreate("test")
             .setSettings(Settings.builder()
+                .put(indexSettings())
                 .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1 + randomInt(2))
                 .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, randomInt(3))
             ));

